airflow/models/dag.py

```diff
@@ -411,15 +411,15 @@ def following_schedule(self, dttm):
             if not self.is_fixed_time_schedule():
                 # relative offset (eg. every 5 minutes)
                 delta = cron.get_next(datetime) - naive
-                following = dttm.in_timezone(self.timezone).add_timedelta(delta)
+                following = dttm.in_timezone(self.timezone).__add__(delta)
```

Comment:
Don't do this -- use of `__add__` or any dunder (double underscore) except `__class__` is a huge code smell.

```suggestion
                following = dttm.in_timezone(self.timezone) + delta
```